requirements/requirements.in

```diff
@@ -35,6 +35,7 @@ openshift
 pexpect==4.7.0 # see library notes
 prometheus_client
 psycopg2
+psycopg
```

Comment:
if _possible_ I would like a comment to state what uses this

Reply:
Can do. For the time being I've been kind of using this branch as my playground for working on this stuff, so I'd say don't review things in this branch super closely right now.

Reply:
Done in 9755825.